recipes/open62541/all/conanfile.py

```diff
@@ -0,0 +1,300 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import glob
+import shutil
+import yaml
+
+
+
+class Open62541Conan(ConanFile):
+    name = "open62541"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Open source implementation of OPC UA (OPC Unified Architecture) aka IEC 62541 licensed under Mozilla Public License v2.0"
+    topics = ("conan", "opcua", "iec62541")
+    homepage = "http://open62541.org"
+    license = "MPL-2.0"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    exports = "submoduledata.yml"
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "multithread": "ANY",                                                   # Level of multithreading (0-99: No Multithreading, 100-199: Thread-Safe API, >=200: Internal Threads
+                                                                                # multithread configuration may be
+                                                                                #       1.0 : False, True
+                                                                                #       1.1 : False, thread-safe, internal-threads or an integer
```

Comment:
How does that integer work? It is the same `10` or `30`? If it is, I would list here only the meaningful values... if every number has a different meaning... well, that's how it is 🤷

Reply:
I have extracted meaningful values here : False, thread-safe, internal-threads
But @madebr made a suggestion where he leaved the possibility to the user to set the integer if he prefers. I think it can make sense.